docs/demo/deployments/spire/k8s-workload-registrar.conf

```diff
@@ -7,5 +7,5 @@ pod_controller = true
 add_svc_dns_names = true
 mode = "crd"
 webhook_enabled = true
-identity_template = "ns/{{.Pod.Namespace}}/sa/{{.Pod.ServiceAccount}}"
```

Comment:
Don't alter the old setting

Reply:
I took the example provided by NSM: https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/spire
I can revert this change